api/migrations/20200924004015-convertAgencyToIssuingAgency.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const RecordTypeEnum = require('../src/utils/constants/misc');
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function(db) {  
+  const mClient = await db.connection.connect(db.connectionString, { native_parser: true });
+
+  try {
+    const nrpti = await mClient.collection('nrpti');
+
+    console.log('**** Renaming ManagementPlan and ManagementPlanLNG agency to issuingAgency ****');
+    await renameAgency(nrpti);
+    console.log('**** Finished renaming agency to issuingAgency ****');
+
+    console.log('**** Updating "Environmental Assessment Office" to "EAO" ****');
+    await updateAcronym(nrpti, 'Environmental Assessment Office', 'EAO');
+    console.log('**** Finished updating "Environmental Assessment Office" to "EAO" ****');
+
+    console.log('**** Deleting empty agency field ****');
+    await deleteEmptyAgency(nrpti);
+    console.log('**** Finished deleting empty agency field ****');
+
+  } catch (error) {
+    console.error(`Migration did not complete. Error processing: ${error.message}`);
+  }
+
+  mClient.close();
+};
+
+exports.down = function(db) {
+  return null;
+};
+
+exports._meta = {
+  "version": 1
+};
+
+async function renameAgency(nrpti) {
+  const schemas = ["ManagementPlan", "ManagementPlanLNG"];
```

Comment:
Looks like "ConstructionPlan" and "ConstructionPlanLNG" also had its property changed.

Reply:
I didn't find any issues with the existing data for ConstructionPlan and ConstructionPlanLNG so they weren't included in that function.

Reply:
Below it looks like the models were changed though.  Did they just not have the 'agency' property on the ones in the DB some how?

Sorry, I could be missing something.

Reply:
Yeah the model had "agency", but the DB had "issuingAgency".  So ConstructionPlan and ConstructionPlanLNG didn't need to be fixed in the DB.